### PR TITLE
Add technical news to all pages #185049817

### PR DESCRIPTION
**Pull request by MLumme**
Task here https://www.pivotaltracker.com/story/show/185049817, probably not on all pages, but then news should now show up on all taxon pages and cms-pages.

### CODE REVIEW

File: projects/laji/src/app/+information/information.module.ts [hunk 1]
```diff
@@ -4,9 +4,9 @@ import { SharedModule } from '../shared/shared.module';
 import { routing } from './information.routes';
 import { InformationComponent } from './information.component';
 import { LajiUiModule } from 'projects/laji-ui/src/public-api';
-
+import { TechnicalNewsModule } from '../shared-modules/technical-news/technical-news.module';
```
Comment (aorin) [inline projects/laji/src/app/+information/information.module.ts:7]: Not a big issue but it looks a little weird that there is no space between the lines `import ...` and `@NgModule ...`

File: projects/laji/src/app/+information/information.component.html [hunk 2]
```diff
@@ -1,8 +1,10 @@
+TEST
```
Comment (aorin) [inline projects/laji/src/app/+information/information.component.html:1]: This should be removed

**Review by aorin — CHANGES_REQUESTED** [review]

File: projects/laji/src/app/+information/information.component.html (hunk 2 above)
Comment (MLumme) [reply projects/laji/src/app/+information/information.component.html]: Both done.

**Review by aorin — APPROVED** [review]
:+1:

**Pull request merged**